var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -310,8 +310,14 @@ class Gromacs(CMakePackage, CudaPackage):
     depends_on("sycl", when="+sycl")
     depends_on("lapack")
     depends_on("blas")
-    depends_on("gcc", when="%oneapi ~intel_provided_gcc")
     depends_on("gcc", when="%intel ~intel_provided_gcc")
+    # TODO this can be expanded to all clang-based compilers once
```

Comment:
@mabraham do you want to add more things here?

Reply:
In a later MR, I think